app/src/main/java/org/systers/mentorship/view/activities/AboutActivity.kt

```diff
@@ -6,21 +6,24 @@ import android.view.KeyEvent
 import android.view.View
 import android.webkit.WebView
 import android.webkit.WebViewClient
-import kotlinx.android.synthetic.main.activity_about.*
 import org.systers.mentorship.R
+import org.systers.mentorship.databinding.ActivityAboutBinding
 
 
 class AboutActivity : AppCompatActivity(), View.OnClickListener {
 
     private var clearHistory = false
 
+    private lateinit var aboutBinding: ActivityAboutBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_about)
+        aboutBinding = ActivityAboutBinding.inflate(layoutInflater)
```

Comment:
It would reduce the boiler plate code if you use kotlin property delegate to initialize the biding object.

Reply:
Ok, will try to make them one liner. Did you mean using lazy?

Reply:
> Ok, will try to make them one liner. Did you mean using lazy?

No not lazy. I found an interesting article on medium

Reply:
Using a library for simple tasks like this would only make the project heavy. I found another approach i will try to implement it in here.